src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/RenderOutputParams/index.tsx

```diff
@@ -27,7 +27,7 @@ export const RenderOutputParams = (): ReactElement | null => {
   }: any = useField('callResults', { subscription: { value: true } })
   const multipleResults = !!method && method.outputs.length > 1
 
-  if (!results) {
+  if (results === null || results === undefined || results === '') {
```

Comment:
`results === null || results === undefined` can be expressed idiomatically as `results == null`.

Reply:
good catch, changed. I always forget about that.